canonical-webteam/practices
# Add Prettier ✨

## Done

Added Prettier to the codebase for consistency to ensure we continue to eat our own dogfood 🐶

## QA

- Pull branch
- Run `jekyll serve`
- View the site at: http://127.0.0.1:4000/practices/
- Run through all pages and verify they display as expected
